### PR TITLE
Add state class so entities can qualify for long term statistics

### PR DESCRIPTION
**Pull request by NickM-27**

### CODE REVIEW

**Comment by neilenns** [comment]
Cool, thanks for the contribution! Can you please open an associated issue for this and describe the change, then link it to this PR?

**Comment by NickM-27** [comment]
Fixes #94

File: src/stateClass.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Neil Enns. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+enum StateClass {
+  MEASUREMENT = "measurement"
```
Comment (neilenns) [inline src/stateClass.ts:7]:
```suggestion
  MEASUREMENT = "measurement",
  TOTAL = "total",
  TOTAL_INCREASING = "total_increasing"
```

Comment (neilenns) [reply src/stateClass.ts:7]: Might as well list all three in the enum just in case in the future one of them is needed.

**Review by neilenns — CHANGES_REQUESTED** [review]

**Review by neilenns — APPROVED** [review]

**Pull request merged**